lib/msal-node/src/config/Configuration.ts

```diff
@@ -83,6 +84,8 @@ export type NodeSystemOptions = {
     networkClient?: INetworkModule;
     proxyUrl?: string;
     customAgentOptions?: http.AgentOptions | https.AgentOptions;
+    tokenRenewalOffsetSeconds?: number;
+    cancellationToken?: number;
```

Comment:
We should not add this new option just for MSI. Cancellation / timeout always existed in MSAL .NET but we are not bringing it in NodeJS as part of MSI. If ppl need it, we can bring it separately. 

Cancellation of async operations in NodeJS is different than cancellation in .NET

Reply:
I mentioned this in a response to the previous comment, but these already exist in the CommonClientConfiguration which all other msal-node clients extend. MSI doesn't extend CommonClientConfiguration, so I added them to the MSI configuration. Thoughts? @sameerag what are your thoughts?

Reply:
Let's start without. We can always add later?